src/content/en/2021/javascript.md

```diff
@@ -364,23 +371,25 @@ When sending resources over the network, it becomes important to look at an effi
 
 ### Compression
 
-Most of the compressed resources use either gzip compression, or brotli (br) compression, or the compression technique is just not set.
+Most of the compressed resources use either gzip compression, brotli (br) compression, or a compression technique that is not set.
+
+{# TODO - should we have links for gzip and brotli? i think other chapters do #}
```

Comment:
@NishuGoel

Reply:
Sure.
Brotli - https://github.com/google/brotli
Gzip - https://www.gnu.org/software/gzip/manual/gzip.html

Could you update please?